src/rpc/blockchain.cpp

```diff
@@ -1520,15 +1589,41 @@ static RPCHelpMan getmempoolinfo()
                         {RPCResult::Type::NUM, "maxmempool", "Maximum memory usage for the mempool"},
                         {RPCResult::Type::STR_AMOUNT, "mempoolminfee", "Minimum fee rate in " + CURRENCY_UNIT + "/kB for tx to be accepted. Is the maximum of minrelaytxfee and minimum mempool fee"},
                         {RPCResult::Type::STR_AMOUNT, "minrelaytxfee", "Current minimum relay fee for transactions"},
-                        {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"}
+                        {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"},
+                        {RPCResult::Type::OBJ, "fee_histogram", "",
+                            {
+                                {RPCResult::Type::OBJ, "<feerate-group>", "Object per feerate group",
+                                {
+                                    {RPCResult::Type::NUM, "sizes", "Cumulated size of all transactions in feerate group"},
+                                    {RPCResult::Type::NUM, "count", "Amount of transactions in feerate group"},
+                                    {RPCResult::Type::NUM, "fees", "Cumulated fee of all transactions in feerate group"},
```

Comment:
Probably both should be "fees" in this sentence, like in line 1603.

Reply:
Fixed, thanks!